requirements.txt

```diff
@@ -15,11 +15,14 @@ pyexcel-xlsx==0.5.8
 retrying==1.3.3
 scipy==0.14.1; python_version < '3.6'
 scipy==1.4.1; python_version > '3.5'
-six
+six~=1.15.0
```

Comment:
Did you add these version constraints in the requirements file? Or did PyCharm add them?
Need to be careful with these, as locking in versions can cause problems wherever a python environment is set up for this repository. I suggest only including changes that are really necessary (e.g. if an older or newer version of a package breaks the code here).

Reply:
BTW, we shouldn't need `six` anymore (that was for compatibility while migrating from Python 2 to 3) - but that's an issue separate from this PR.

Reply:
I think all the changes made in requirements were suggested by PyCharm and I accepted them, so I imagine they were necessary. Not sure what to do about it.